### PR TITLE
Minor code fixes

### PR DESCRIPTION
**Pull request by mayanksuman**
Minor code changes based on suggestion from CodeQL and QT creator.

### CODE REVIEW

**Comment by mayanksuman** [comment]
@jahnf , the `continueOnError` argument is not being used in function `SubHidppConnection::sendDataBatch` and `SubHidppConnection::sendRequestBatch`.

**Comment by jahnf** [comment]
> @jahnf , the `continueOnError` argument is not being used in function `SubHidppConnection::sendDataBatch` and `SubHidppConnection::sendRequestBatch`.

It is used right immediately in the postSelf call (e.g. line 204)

**Comment by jahnf** [comment]
> > @jahnf , the `continueOnError` argument is not being used in function `SubHidppConnection::sendDataBatch` and `SubHidppConnection::sendRequestBatch`.
> 
> It is used right immediately in the postSelf call (e.g. line 204)

Ah I see, its duplicated..

File: src/projecteurapp.cc [hunk 1]
```diff
@@ -268,7 +268,7 @@ ProjecteurApplication::ProjecteurApplication(int &argc, char **argv, const Optio
 
   // Open local server for local IPC commands, e.g. from other command line instances
   QLocalServer::removeServer(localServerName());
-  if (m_localServer->listen(localServerName()))
+  if (m_localServer && m_localServer->listen(localServerName()))
```
Comment (jahnf) [inline src/projecteurapp.cc:271]: `m_localServer` is instantiated in the constructor. If that fails (memory allocation error) the program will fail long before reaching this point.

File: src/projecteurapp.cc [hunk 2]
```diff
@@ -523,9 +523,7 @@ void ProjecteurApplication::setCurrentCursorPos(const QPoint& pos)
 void ProjecteurApplication::readCommand(QLocalSocket* clientConnection)
 {
   auto it = m_commandConnections.find(clientConnection);
-  if (it == m_commandConnections.end()) {
-    return;
-  }
+  if (it == m_commandConnections.end()) return;
```
Comment (jahnf) [inline src/projecteurapp.cc:526]: I know that I am not consistent myself, but I like to put code after if clauses in braces.

Comment (mayanksuman) [reply src/projecteurapp.cc:526]: Ok. I will update that one in next commit.

File: src/projecteurapp.cc (hunk 1 above)
Comment (mayanksuman) [reply src/projecteurapp.cc:271]: Yes, I saw that, but, just to turn off warning I have introduced extra check for `m_localServer`. However, it is not necessary. Will remove it in next commit.

**Comment by mayanksuman** [comment]
The suggested reviews are now implemented.

**Pull request merged**